syntax = "proto3";
package duchess.v1alpha1;

import "duchess/v1alpha1/games.proto";
import "duchess/v1alpha1/positions.proto";


// Duchess Backend API
service DuchessBackendAPI {
    // Fetch a collection of games
    rpc GetSimilarGames(GetSimilarGamesRequest) returns (GetSimilarGamesResponse);
}
